code/routers/country_page/country.js

```diff
@@ -99,28 +99,192 @@ router.get('/get-country-info', (req, res) => {
+    var countryQuery = {name: req.body[0].countryName}; 
+  }else{
+    var orgQuery = {orgName: req.body.orgName}; 
+    var countryQuery = {name: req.body.countryName};  
+  }
+  
+
+  Organization.find(orgQuery, function(err, organizations) {
+    if(err || organizations.length == 0){
+      return res.status(countryCodes.insertOrganizations.organizationsNotFound.status).send({
+        message: countryCodes.insertOrganizations.organizationsNotFound.message
       });
+    }else{
+      var updatedOrgIDs = []
+      Country.find(countryQuery, function(err, country) {
+        if(err) {
+          return res.status(countryCodes.insertOrganizations.countryNotFound.status).send({
+            message: countryCodes.insertOrganizations.countryNotFound.message
+          });
+        }else{
+          var prevOrgs = country[0].organizationIDs;
+          if(Array.isArray(prevOrgs)){
+            for (var i = 0; i < prevOrgs.length; i++) {
+              updatedOrgIDs.push(prevOrgs[i]);
+            }
+          }
+          for (var i = 0; i < organizations.length; i++) {
+            updatedOrgIDs.push(organizations[i]._id);
+          }
+          
+          var updateQuery = {organizationIDs: updatedOrgIDs};
+          Country.findOneAndUpdate(countryQuery, updateQuery, {upsert: true}, function(err, country) {
+            if(err) {
+              return res.status(countryCodes.insertOrganizations.organizationUpdateFail.status).send({
+                message: countryCodes.insertOrganizations.organizationUpdateFail.message
+              });
+            }else{
+              return res.status(countryCodes.insertOrganizations.success.status).send({
+                message: countryCodes.insertOrganizations.success.message
+              });
+            }
+          })
+        }
+      })
     }
   })
 })
 
 
+/*
+    Remove Organizations API
+    inputs:
+      [{
+      countryName: String,
+      orgName: String
+      }]
+
+    Expects all countryNames to be the same.
+*/
+router.post('/remove-organizations', (req, res) => {
+  
+  //Handling of multiple organizations, still expecting them to be from the same country
+  if(Array.isArray(req.body)){
+    var orgNames = [];
+    for (var i = 0; i < req.body.length; i++){
+      var curBody = req.body[i];
+      orgNames.push(curBody.orgName);
+    }
+    var orgQuery = {orgName: {$in: orgNames}};
+    var countryQuery = {name: req.body[0].countryName}; 
+  }else{
+    var orgQuery = {orgName: req.body.orgName}; 
+    var countryQuery = {name: req.body.countryName};  
+  }
+
+  Organization.find(orgQuery, function(err, organizations) {
+    if(err || organizations.length == 0){
+      return res.status(countryCodes.removeOrganizations.organizationsNotFound.status).send({
+        message: countryCodes.removeOrganizations.organizationsNotFound.message
+      });
+    }else{
+      var updatedOrgIDs = []
+      Country.find(countryQuery, function(err, country) {
+        if(err) {
```

Comment:
This should also check if country length > 0 or else line 250 can cause an error